lib/validators/user.schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "type": "object",
+  "properties": {
+    "login": {
+      "type": "string"
+    },
+    "password": {
+      "type": "string",
+      "pattern": "^(?=.*?\\d)(?=.*?[a-zA-Z])[a-zA-Z\\d]+$"
+    },
+    "age": {
+      "type": "integer",
+      "minimum": 4,
+      "maximum": 130
+    },
+    "isDeleted": {
```

Comment:
isDeleted flag is an internal information and should not be present in payload.

Reply:
Done